### PR TITLE
WIP: adding tests

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by iprotsyuk** [comment]
:+1:

**Comment by antgonza** [comment]
@eldeveloper, can you review this? 

@iprotsyuk @eldeveloper, I think it will be worth adding merging this one so we can start solving errors and really testing the code; what do you think?

**Comment by iprotsyuk** [comment]
@antgonza, agree

File: tests/javascript_tests/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!--
+/**
+ *
+ * @author Yoshiki Vazquez Baeza
+ * @copyright Copyright 2013, The ili Project
+ * @credits Yoshiki Vazquez Baeza
+ * @license BSD
+ * @version 0.9.3-dev
```
Comment (ElDeveloper) [inline tests/javascript_tests/index.html:8]: Can you remove this whole block altogether?

File: tests/javascript_tests/index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <!-- Testing framework requirements -->
+  <script src="qunit/qunit-min.js"></script>
+
+  <!-- Qunit depends on jQuery -->
+  <script src="../../js/lib/jquery-2.1.4.min.js"></script>
+
+  <!-- Libraries the code depends on -->
+
+  <!--Source files to test-->
+  <script src="../../js/ColorMaps.js"></script>
```
Comment (ElDeveloper) [inline tests/javascript_tests/index.html:36]: This is fine as is, but just want to note that once we finish require-fying we are going to need to update this to use require.

**Comment by ElDeveloper** [comment]
👍

**Pull request merged**